lib/ledger_sync/adaptors/quickbooks_online/operation/find.rb

```diff
@@ -12,11 +12,11 @@ class Find
           private
 
           def operate
-            return failure(
-              Errors::OperationError::LedgerIDRequired.new(
-                operation: self
-              )
-            ) if resource.ledger_id.nil?
+            # return failure(
+            #   Error::OperationError::LedgerIDRequired.new(
+            #     operation: self
+            #   )
+            # ) if resource.ledger_id.nil?
```

Comment:
This had a typo for error. Anyway, do we need this check in a first place? when all our ledger operations defines `required(:ledger_id).filled(:string)`

Reply:
I suppose it's redundant.  If it is only used here, can we just nuke any mention/definition of it?